File: QuanLyPhongKham.Server/src/QuanLyPhongKham.DTOs/Responses/HoaDonResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QuanLyPhongKham.DTOs.Responses
{
    public class HoaDonResponse
    {
        public Guid MaHD { get; set; }
        public DateTime NgayThanhToan { get; set; }
        public decimal TongTien { get; set; }
        public string? TrangThaiThanhToan { get; set; }
        public string? MaLeTan { get; set; }
        public Guid? MaPK { get; set; }
        public string? TenBenhNhan { get; set; }
    }
}

File: QuanLyPhongKham.Client/src/interfaces/responses/HoaDonResponse.js
/**
 * @typedef {Object} HoaDonResponse
 * @property {string} maHD
 * @property {string|Date} ngayThanhToan
 * @property {number} tongTien
 * @property {string} trangThaiThanhToan
 * @property {string} maLeTan
 * @property {string} maPK
 * @property {string} tenBenhNhan
 */

export const createHoaDonResponse = (
	maHD = "",
	ngayThanhToan = null,
	tongTien = 0,
	trangThaiThanhToan = "",
	maLeTan = "",
	maPK = "",
	tenBenhNhan = "",
) => ({
	maHD,
	ngayThanhToan,
	tongTien,
	trangThaiThanhToan,
	maLeTan,
	maPK,
	tenBenhNhan,
});
